tardis/conftest.py

```diff
@@ -32,8 +51,50 @@ def pytest_addoption(parser):
     parser.addoption("--atomic-dataset", dest='atomic-dataset', default=None,
                      help="filename for atomic dataset")
 
-    parser.addoption("--slow-test-data", dest="slow-test-data",
-                     help="path to directory having baseline data for slow tests")
+    parser.addoption("--integration-tests", dest="integration-tests",
+                     help="path to configuration file for integration tests")
+
+
+def pytest_configure(config):
+    html_file = tempfile.NamedTemporaryFile(delete=False)
+    # Html test report will be generated at this filepath by pytest-html plugin
+    config.option.htmlpath = html_file.name
+
+
+def pytest_unconfigure(config):
+    # Html report created by pytest-html plugin is read here, uploaded to
+    # dokuwiki and finally deleted.
+    if dokuwiki_available:
+        githash = tardis.__githash__
+        report_content = open(config.option.htmlpath, 'rb').read()
+        report_content = report_content.replace("<!DOCTYPE html>", "")
+
+        report_content = (
+            "Test executed on commit "
+            "[[https://www.github.com/tardis-sn/tardis/commit/{0}|{0}]]\n\n"
+            "{1}".format(githash, report_content)
+        )
```

Comment:
@wkerzendorf These four lines are good to be here or in `else` block ? I spoke to myself: _try to make a connection, except these two errors, else upload the report we have_. Hence, I kept the report ready beforehand (compromising the time of execution of these lines if connection isn't at all made.)